include/sparrow/typed_array.hpp

```diff
@@ -63,6 +69,8 @@ namespace sparrow
         using const_bitmap_range = typename layout_type::const_bitmap_range;
         using const_value_range = typename layout_type::const_value_range;
 
+        typed_array();
```

Comment:
Suggestion if possible: make this `= default;` and set the default values of the members in their declaration line.

Reply:
Done